UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
Hehe indeed 😆 
Will merge as soon as Travis validates the PR 😉 

Great job anway, thanks for the awesome work! 👍
_(I'm looking forward to see where we'll go from here to improve it even better in future versions 😉 )_